js/core/component.js

```diff
@@ -490,6 +456,23 @@ var Component = Class.inherit({
         return result;
     },
 
+    _convertToObj(options, value) {
```

Comment:
Better make this a static function outside the class (no need to use `this` context)

Reply:
What about renaming it to smth like `normalizeOptions` or `normalizeOptionsObj` ?